content/docs/use-cases/ci-and-ml.md

```diff
@@ -0,0 +1,171 @@
+# CI for ML
+
+## Motivation
+
+ML engineers and data scientists face many unique challenges. Some of these
+challenges can be solved by adopting first-citizen concepts from traditional
```

Comment:
What are "first-citizen concepts"? Not familiar with the phrase 🙂 is it similar to "first-class citizen"?

Reply:
😲 True! I'll replace it with: "... by adopting well-established concepts from...". @jorgeorpinel what do you think?

Reply:
Sure, I honestly thought maybe it was a phrase I didn't know. If it's a typo feel free to replace it with something else like the above example. Consider term "best practices" maybe.